integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/DDLStrategy.scala

```diff
@@ -193,6 +197,32 @@ class DDLStrategy(sparkSession: SparkSession) extends SparkStrategy {
         ExecutedCommandExec(
           CarbonAlterTableUnsetCommand(tableName, propKeys, ifExists, isView)) :: Nil
       }
+      case RefreshTable(tableIdentifier) =>
+        val isCarbonTable = CarbonEnv.getInstance(sparkSession).carbonMetastore
```

Comment:
here just call `RefreshCarbonTableCommand ` and at the end of that command you can call `RefreshTable`

Reply:
Fixed